symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/properties/CorsProperties.java

```diff
@@ -0,0 +1,22 @@
+package com.symphony.bdk.app.spring.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Configuration Properties for enabling the CORS support to accept requests coming from the extension app.
+ */
+@Getter
+@Setter
+public class CorsProperties {
+
+  /**
+   * CORS handling for the specified path pattern.
+   */
+  private String allowedOrigin = "/**";
+
+  /**
+   * The list of allowed origins pattern.
+   */
+  private String urlMapping = "*";
```

Comment:
Same here?

Reply:
I'm not sure how to handle multiple `urlMapping`. Is it valid?
```
registry.addMapping(url1).allowedOrigins(String ...)
registry.addMapping(url2).allowedOrigins(String ...)
```

Reply:
I think that `registry` is a builder, so it would be possible to: 
```java
for(String mapping : mappings) { 
   registry.addMapping(mapping);
}
registry.allowedOrigins(String ...)
```

Reply:
Ok it's not working. Then we need to follow how https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/servlet/config/annotation/CorsRegistry.html is designed: 
```yaml
bdk-app:
  cors: 
    '/**': 
      allowedOrigins: 
        - '*'
      allowedHeaders: 
        - '*'
```